contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -1,24 +1,34 @@
 # Volume Topology-aware Scheduling
 
-Authors: @msau42
+Authors: @msau42, @lichuqiang
 
 This document presents a detailed design for making the default Kubernetes
 scheduler aware of volume topology constraints, and making the
 PersistentVolumeClaim (PVC) binding aware of scheduling decisions.
 
+## Definitions
+* Topology: Rules to describe accessibility of an object with respect to
+  location in a cluster.
+* Domain: A grouping of locations within a cluster. For example, 'node1',
+  'rack10', 'zone5'.
+* Topology Key: A description of a general class of domains. For example,
+  'node', 'rack', 'zone'.
+* Hierarchical domain: Domain that can be fully encompassed in a larger domain.
+  For example, the 'zone1' domain can be fully encompassed in the 'region1'
+  domain.
+* Failover domain: A domain that a workload intends to run in at a later time.
 
 ## Goals
-* Allow a Pod to request one or more topology-constrained Persistent
-Volumes (PV) that are compatible with the Pod's other scheduling
-constraints, such as resource requirements and affinity/anti-affinity
-policies.
-* Support arbitrary PV topology constraints (i.e. node,
-rack, zone, foo, bar).
-* Support topology constraints for statically created PVs and dynamically
-provisioned PVs.
+* Allow the Kubernetes scheduler to influence where a volume is provisioned or
+  which pre-existing volume to bind to based on scheduling constraints on the
+  Pod requesting a volume, such as Pod resource requirements and
+  affinity/anti-affinity policies.
+* Support arbitrary PV topology domains (i.e. node, rack, zone, foo, bar)
+  without encoding each as first class objects in the Kubernetes API.
+* Support topology for pre-provisioned PVs and dynamically provisioned PVs.
```

Comment:
nit: Maybe expand on `Support topology` to `Enable scheduler to use topology information of referenced pre-provisioned or dynamically provisioned PVs to...`?